Kudu.Core/Environment.cs

```diff
@@ -106,8 +106,8 @@ public class Environment : IEnvironment
             {
                 // in linux, rootPath is "/home", while .ssh folder need to under "/home/{user}", 
                 // and username and site name is always the same in actual deployment
-                string siteName = System.Environment.GetEnvironmentVariable("APP_POOL_ID");
-                siteName = NormalizeAppPoolId(siteName);
+                string siteName = System.Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME");
```

Comment:
Need to revert back to use "WEBSITE_SITE_NAME". On mono LWAS pass site name via "WEBSITE_SITE_NAME" environment valuable. We do not have "APP_POOL_ID"